bpf/bpf_lxc.c

```diff
@@ -1075,7 +1061,7 @@ ipv6_policy(struct __ctx_buff *ctx, int ifindex, __u32 src_label, __u8 *reason,
 		send_trace_notify6(ctx, TRACE_TO_PROXY, src_label, SECLABEL, &orig_sip,
 				  0, ifindex, 0, monitor);
 		if (tuple_out)
-			memcpy(tuple_out, &tuple, sizeof(tuple));
+			*tuple_out = tuple;
```

Comment:
Could it be that the difference here was intentional due to the different sizes between v4 and v6 tuples?

Reply:
OK, reverted back.